boards/arm/nrf51_vbluno51/nrf51_vbluno51_defconfig

```diff
@@ -7,6 +7,7 @@ CONFIG_BOARD_NRF51_VBLUNO51=y
 # enable uart driver
 CONFIG_SERIAL=y
 CONFIG_UART_NRF5=y
+CONFIG_UART_NRF5_FLOW_CONTROL=n
```

Comment:
if your board has no CTS/RTS line connections, then flow control should be disabled in the board Kconfig. Disabling in defconfig will still permit applications to enable it, which I assume is not what you desire.

Reply:
Hi @cvinayak , The VBLUno51 board has CTS/RTS line connections. Because using UART hardware flow control features depend on DAPLink firmware version, so the initial default it needs to be disabled.

Reply:
@iotvietmember If you check drivers/serial/Kconfig.nrf5  the default is already n for UART_NRF5_FLOW_CONTROL.
Only samples/bluetooth/hci_uart/nrf5.conf has it enabled. Hence, you PR is not needed.

Reply:
Yeah, I understood. Many thanks